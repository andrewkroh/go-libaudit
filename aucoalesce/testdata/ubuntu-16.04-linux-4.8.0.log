type=SERVICE_START msg=audit(1492625990.183:4083): pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=rsyslog comm="systemd" exe="/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'