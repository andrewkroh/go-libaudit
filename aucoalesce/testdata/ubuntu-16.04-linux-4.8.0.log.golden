{
  "@timestamp": "2017-04-19T18:19:50.183Z",
  "sequence": 4083,
  "category": "system-services",
  "record_type": "service_start",
  "result": "success",
  "session": "unset",
  "actor": {
    "primary": "unset",
    "secondary": "0"
  },
  "thing": {},
  "data": {
    "comm": "systemd",
    "exe": "/lib/systemd/systemd",
    "pid": "1",
    "unit": "rsyslog"
  }
}
